bin/hive_setup

```diff
@@ -212,8 +212,8 @@ def setup_hive(options)
     f.puts ''
     f.puts '  # Configuration for various network options'
     f.puts '  network:'
-    f.puts '    scheduler: http://localhost:3000'
-    f.puts '    #hive_mind: http://localhost:3001'
+    f.puts "    scheduler: <%=ENV['HIVE_CI_URL'] || 'http://localhost:3000' %>"
+    f.puts "    #hive_mind: <%=ENV['HIVE_MIND_URL'] || 'http://localhost:3001' %>"
```

Comment:
<%=ENV['ABC'] || 'http://something' %>" will not work. I think this works only with erb files.

<img width="504" alt="screen shot 2017-08-07 at 10 36 27" src="https://user-images.githubusercontent.com/8381766/29021159-7f91fccc-7b5c-11e7-98a3-d2d6b3635e17.png">